core/imageroot/var/lib/nethserver/node/actions/remove-module/50update

```diff
@@ -42,6 +42,9 @@ if [[ -z $uid ]]; then # Rootfull module
 else # Rootless module
     loginctl disable-linger "${mid}"
     systemctl stop "user@${uid}.service"
+    if pgrep -u "${mid}"; then
+        loginctl terminate-user "${mid}"
+    fi
```

Comment:
Ignore non-existing user error

```suggestion
    loginctl terminate-user "${mid}" &>/dev/null || :
```

Reply:
can we go to 

`loginctl terminate-user "${mid}" &>/dev/null || true`

Reply:
:shrug: They are equivalent, they both set the expression exit code to 0 overriding `set -e` behavior.

https://www.gnu.org/software/bash/manual/html_node/Bourne-Shell-Builtins.html#index-_003a

Reply:
lets go to your proposal then.